spring-swagger-codegen/spring-openapi-generator-api-client/.gitignore

```diff
@@ -0,0 +1,21 @@
+*.class
```

Comment:
Question: does the OpenAPI generator also generate the .gitignore file? If so, keep it, otherwise we don't need it because the one at the root of the repo will take care of things.

Reply:
Yes it does.  I'll remove it.

Reply:
Removed